app/assets/javascripts/search.js

```diff
@@ -5,18 +5,21 @@ if($("#advanced-search").length){
   var $description = $('input#description');
   var $downloads   = $('input#downloads');
   var $updated     = $('input#updated');
+  var $yanked      = $('input#yanked');
 
-  $name.add($summary)
+    $name.add($summary)
     .add($description)
     .add($downloads)
     .add($updated)
+    .add($yanked)
     .on('input', function(e) {
       var name        = $name.val().length > 0 ? 'name: ' + $name.val() : '';
       var summary     = $summary.val().length > 0 ? 'summary: ' + $summary.val() : '';
       var description = $description.val().length > 0 ? 'description: ' + $description.val() : '';
       var downloads   = $downloads.val().length > 0 ? 'downloads: ' + $downloads.val() : '';
       var updated     = $updated.val().length > 0 ? 'updated: ' + $updated.val() : '';
+      var yanked      = $yanked.is(":checked") ? 'yanked: true' : '';
```

Comment:
We would like to show count of yanked gems here. I think we need to add an aggregation like [this](https://github.com/rubygems/rubygems.org/blob/master/lib/elastic_searcher.rb#L47) with filter `yanked: true`.

Reply:
Hey, so as discussed above, we will show yanked gems, if a filter is checked. So if user is not shown any yanked gems in normal result but is given a filter with count of yanked gem will make it improper imo. Because currrently, the counts of filters are out of the shown results only.
Also if the user has selected the yanked filter, all the shown gems will be yanked, show again filter doesn't make sense.

Can you clarify this? @sonalkr132 :sweat_smile:

Reply:
I don't think filters have to co-related with results on search page. Consider example of github search, it shows search result of code by default, however if you click on filter of `issues`, it will show results only for that filter.
Issues, commits etc are not part of default search result page but their count is helpful info.
Perhaps I can better understand your concern if you can further explain why you believe this could be an issue.